src/chatlog/chatlog.cpp

```diff
@@ -456,11 +456,18 @@ void ChatLog::startResizeWorker()
             workerAnchorLine = visibleLines.first();
     }
 
+    // switch to busy scene displaying the busy notification if there is a lot
+    // of text to be resized
+    int txt = 0;
+    for(ChatLine::Ptr line : lines)
```

Comment:
I had thought of it, and so i calculated time taken for counting for several length of chatlog.
```
    long long t =  QDateTime::currentMSecsSinceEpoch();
    int txt = 0;
    for(ChatLine::Ptr line : lines)
        for(ChatLineContent* content : line->content)
            txt += content->getText().size();
    qDebug()<<(QDateTime::currentMSecsSinceEpoch()-t)<<" txt="<<txt;
    if(txt>500000)
        setScene(busyScene);
```
and the results were very good. When resizing took 5 seconds, the time taken for counting was 0 milliseconds.
This was the qDebug message i got.
<code>0  txt= 2378625</code>
I never got time taken more than 0 millisecons.